examples/gno.land/p/demo/ownable/ownable_test.gno

```diff
@@ -67,9 +60,7 @@ func TestCallerIsOwner(t *testing.T) {
 	std.TestSetOrigCaller(unauthorizedCaller) // TODO(bug): should not be needed
 
 	err := o.CallerIsOwner()
-	if err == nil {
-		t.Fatalf("Expected %s to not be owner", unauthorizedCaller)
-	}
+	uassert.Error(t, err) // XXX: IsError(..., unauthorizedCaller)
```

Comment:
Why implementing IsError is not possible?

Reply:
The base go implementation of this uses reflection 🙁

Reply:
we can do ErrorContains(t, err, myerr.Error()) which is using strings.Contains.

Reply:
The previous test was simply checking that the error is not equal to nil, which is exactly what `uassert.Error` is doing. My comment is more of a suggestion to improve the unit tests in the future (comparing error type, not just non-nil). However, I believe we are equally good or better at testing code as before with this PR.